### PR TITLE
fix: return member data on kick and ban event

### PR DESCRIPTION
**Pull request by Doacola**
This should fix the returned member data for #87

### CODE REVIEW

**Comment by Scraayp** [comment]
Checked all good to go for merge

**Comment by codsane** [comment]
Looks good from here as well

File: index.js [hunk 1]
```diff
@@ -218,6 +218,7 @@ class AntiSpam extends EventEmitter {
 			}
 
 			try {
+				const tempMember = message.member
```
Comment (Androz2091) [inline index.js:221]: Add missing semicolon 😅 

```suggestion
				const tempMember = message.member;
```

Comment (Scraayp) [reply index.js]: Wow

**Pull request merged**